src/Interface/Modules/Render/Ospray/OSPRayRenderer.cpp

```diff
@@ -46,7 +46,7 @@ using namespace Render;
 using namespace Core::Datatypes;
 using namespace Core::Geometry;
 
-#ifdef WITH_OSPRAY
+#if defined(WITH_OSPRAY) || defined(WITH_OSPRAY_LOCAL)
```

Comment:
Can you define this disjunction in CMake? Then no C++ files need to change.

WITH_OSPRAY --> BUILD_OSPRAY
WITH_OSPRAY_LOCAL --> PREBUILT_OSPRAY
Then add cmake for WITH_OSPRAY = BUILD_OSPRAY || PREBUILT_OSPRAY
A few cmake files will need to change, but no C++

Reply:
Good idea, I'll get this changed

Reply:
@tarkpate I could handle this change if you are too busy.

Reply:
Just updated with these new flags